pyqmc/orbitals.py

```diff
@@ -185,3 +190,24 @@ def pgradient(self, ao, spin):
 
         """
         return self.param_split[spin], ao
+
+    def _estimate_rcut(self, cell):
```

Comment:
This doesn't need to be a class member (doesn't use self).

Make precision an argument.  (cell, precision)